portfolio.html

```diff
@@ -18,7 +19,24 @@
     </nav>
   </header>
   <main>
+    <table width="400" cellpadding="10" cellspacing="5"
```

Comment:
Tables are only for data, not layout. Please adjust this code to use valid html.

Reply:
ok, thanks. i used an unordered list instead.